### PR TITLE
Release-1.10.3-Changelog

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Review by swathinsankaran — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -375,3 +375,10 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - Fix: AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
  - Fix: CTRL_CA_DATA env variable, from AKO Statefulset definition, violates CNTR-K8-001160.
+
+## AKO-1.10.3
```
Comment (swathinsankaran) [inline CHANGELOG.md:379]: should we add the new field `networkUUID` in `values.yaml` under `Added`?

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: CHANGELOG.md (hunk 1 above)
Comment (akshayhavile) [reply CHANGELOG.md:379]: Addressed

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Pull request merged**